### DISCUSSION TITLE
Semi transparent sprite or shadow

### DISCUSSION THREAD

**Discussion by AcuarioCat** [Bodmer/TFT_eSPI#2667] [Q&A - Sprites]
Is it possible to have a semi-transparent sprite where when one sprite with transparency is on top of another sprite the transparent part causes a shadow on the sprite below but the content of the sprite below can still be seen?

**Comment by Bodmer** [comment]
The library does not have a function for that although it could be implemented with the alphaBlend function.

It is not clear whether you want to simply dim parts of a sprite (i.e. have a  mask) or whether you want to overlay one image with an alpha channel on top of another image.
